web/react/components/posts_view.jsx

```diff
@@ -38,6 +40,15 @@ export default class PostsView extends React.Component {
     static get SCROLL_TYPE_POST() {
         return 5;
     }
+    forceUpdateInfo() {
```

Comment:
Soon all `refs` will point directly to the DOM nodes and you won't be able to `forceUpdate()` like this. What you can do instead is add a prop to posts called `displayType` or `displayNameType` that will change based on the setting in the preference store. Changing that prop will cause the posts to re-render.